src/Race.js

```diff
@@ -0,0 +1,34 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
+class Race {
+  static MOVEABLE_NUMBER = 4;
+
+  static runRaceRounds(cars, attemptCount) {
+    Console.print('\n실행 결과');
+    for (let i = 0; i < attemptCount; i += 1) {
+      this.moveCar(cars);
+      this.printCarProgress(cars);
+    }
+  }
+
+  static printCarProgress(cars) {
+    cars.forEach(car => Console.print(`${car.name} : ${'-'.repeat(car.move)}`));
```

Comment:
```suggestion
    cars.forEach(({name,move}) => Console.print(`${name} : ${'-'.repeat(move)}`));
```
구조분해를 이용하면 더 깔끔하게 쓸 수 있을 것 같아요!! 👍